packages/app/src/specs/InlineSpecListHeader.vue

```diff
@@ -60,26 +50,36 @@
         />
       </button>
     </div>
-    <button
-      class="
-        rounded-md flex
-        outline-none
-        border-1
-        border-gray-900
-        h-24px
-        w-24px
-        duration-300
-        hocus-default
-        items-center
-        justify-center
-        hocus:ring-0
-        hocus:border-indigo-300
-      "
-      :aria-label="t('specPage.newSpecButton')"
-      @click="emit('newSpec')"
+    <Tooltip
+      placement="right"
+      data-cy="tooltip"
     >
-      <i-cy-add-small_x16 class="icon-light-gray-50 icon-dark-gray-200" />
-    </button>
+      <button
+        class="rounded-md flex outline-none border-1 border-gray-900 h-24px w-24px duration-300 hocus-default items-center justify-center hocus:ring-0 hocus:border-indigo-300"
+        :aria-label="t('specPage.newSpecButton')"
+        @click="emit('newSpec')"
+      >
+        <i-cy-add-small_x16 class="icon-light-gray-50 icon-dark-gray-200" />
+      </button>
+      <template
+        #popper
+      >
+        <span
+          class="font-normal text-sm inline-flex"
+          data-cy="tooltip-content"
+        >
+          {{ t('specPage.newSpecButton') }}
+        </span>
+      </template>
+    </Tooltip>
+    <InlineRunAllSpecs
+      v-if="isRunAllSpecsAllowed"
+      spec-number="all"
+      directory="all"
+      grayscale
+      class="rounded-md flex outline-none border-1 border-gray-900 h-24px w-24px duration-300 hocus-default items-center justify-center hocus:ring-0 hocus:border-indigo-300"
+      @runAllSpecs="emit('runAllSpecs')"
```

Comment:
It looks like we just emit up to the top then do `store.runAllSpecs` - and particular reason to emit up then call the method (as opposed to just importing the store and calling the method inside of the `InlineRunAllSpecs` component)? Or was that to make it more testable?